packages/grid/_modules_/grid/hooks/features/keyboard/useGridKeyboard.ts

```diff
@@ -212,7 +212,8 @@ export const useGridKeyboard = (
         return;
       }
 
-      if (isEnterKey(event.key)) {
+      if (isEnterKey(event.key) || isSpaceKey(event.key)) {
+        event.preventDefault();
```

Comment:
We should only prevent default if space, enter doesn't need it.

Reply:
fixed